react-function-components/src/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+
+ReactDOM.render(CustomButton(), document.querySelector('#root'));
```

Comment:
In react, we never really call our own components. Instead, create a React element.

```jsx
ReactDOM.render(
  <CustomButton />,
  document.querySelector('#root')
);
```